### ISSUE TITLE
Any chance of extending support to circle ci?

### ISSUE THREAD

**Issue by goanpeca** [grosser/travis_dedup#8]

**Comment by grosser** [comment]
Yep, just need api docs for circle-ci and a brave hero like @goanpeca to open a PR ;)

**Comment by goanpeca** [comment]
Ok, then I will get to learning ruby now :-|... if only it was written with python...:sigh:

**Comment by grosser** [comment]
hmm they are not that different ;)

you could start by finding the api endpoints this would need to hit and making an example app to test against ... rest should hopefull be mostly copy-paste with a few renames

**Comment by goanpeca** [comment]
Yep, will take a look over the weekend :-), thanks
